torch/optim/lr_scheduler.py

```diff
@@ -1992,8 +1994,8 @@ def get_lr(self) -> list[float | Tensor]:
         Computes learning rates for the optimizer's :attr:`~torch.optim.Optimizer.param_groups` following:
 
         .. math::
-            \texttt{eta_min} + \frac{1}{2}(\texttt{base_lr} - \texttt{eta_min})\left(1 +
-            \cos\left(\frac{\texttt{T_cur}}{\texttt{T_i}}\pi\right)\right)
+            \texttt{eta\_min} + \frac{1}{2}(\texttt{base\_lr} - \texttt{eta\_min})\left(1 +
+            \cos\left(\pi \cdot \frac{\texttt{T\_cur}}{\texttt{T\_i}}\right)\right)
```

Comment:
It was the underscore that was the issue? Dang

Reply:
This is surprising cuz the underscores were there ebfore!

Reply:
The underscores that were there in class-level `CosineAnnealingLR` docstring were fine - mine were problematic because they're in the `\texttt` environment which has some quirks in KaTeX (used by Sphinx).

Reply:
Okay, now seeing that even warnings cause docs CI to fail. I should be able to figure these out - I'll get the docs working locally next time so we can avoid the back-and-forth. Sorry!

Reply:
Ping me when it's ready or if you have any qs on how I can help! I do think our docs build is a bit finnicky to get right locally.